cmake/external/openvino.cmake

```diff
@@ -0,0 +1,108 @@
+# Copyright(C) 2019 Intel Corporation
+# Licensed under the MIT License
+
+include (ExternalProject)
+
+set(ngraph_ROOT_DIR ${CMAKE_CURRENT_BINARY_DIR}/external/ngraph)
+set(ngraph_INSTALL_DIR ${ngraph_ROOT_DIR})
+set(ngraph_INCLUDE_DIRS ${ngraph_INSTALL_DIR}/include)
+set(ngraph_LIBRARIES ${ngraph_INSTALL_DIR}/${CMAKE_INSTALL_LIBDIR})
+set(ngraph_SRC ${CMAKE_CURRENT_BINARY_DIR}/ngraph/src/project_ngraph)
+set(prebuilt_ONNX_SOURCE_DIR "${PROJECT_SOURCE_DIR}/external/onnx")
+set(prebuilt_ONNX_BINARY_DIR "${CMAKE_CURRENT_BINARY_DIR}")
+set(ngraph_URL "https://github.com/NervanaSystems/ngraph.git")
+set(ngraph_branch "origin/rov.2020.2")
```

Comment:
for cmake external project, we can't use a branch name, otherwise the build fails.
it is trying to do a git stash but git user is not setup on our CI machines.
either use a tag or commit id.

Reply:
using SHA commit id now